### PR TITLE
Return execution time when wait for command

### PR DESCRIPTION
**Pull request by ZongHong-Lyu**
So it can get more precise execution time (without wait time in queue)

### CODE REVIEW

**Comment by ZongHong-Lyu** [comment]
I've updated the implementation as you said to retain backward compatibility.

File: src/dmp_dv.cpp [hunk 1]
```diff
@@ -256,17 +256,21 @@ int64_t dmp_dv_cmdlist_exec(dmp_dv_cmdlist cmdlist) {
...
   }
-  uint64_t temp_exec_time;
-  int error = ((CDMPDVCmdList*)cmdlist)->Wait(exec_id, temp_exec_time);
-  if (!error && exec_time) {
-    *exec_time = temp_exec_time;
+  return ((CDMPDVCmdList*)cmdlist)->Wait(exec_id);
+}
+
+
+uint64_t dmp_dv_get_last_cmdlist_exec_time(dmp_dv_cmdlist cmdlist) {
```
Comment (ajkxyz) [inline src/dmp_dv.cpp]: Please rename to `dmp_dv_cmdlist_get_last_exec_time`

Comment (ZongHong-Lyu) [reply src/dmp_dv.cpp]: Fixed.

File: include/dmp_dv.h [hunk 2]
```diff
@@ -239,6 +239,12 @@ int64_t dmp_dv_cmdlist_exec(dmp_dv_cmdlist cmdlist);
 int dmp_dv_cmdlist_wait(dmp_dv_cmdlist cmdlist, int64_t exec_id);
 
 
+/// @brief Get the last execution time in microseconds of specified command.
+/// @param cmdlist Handle to command list, when NULL the error is returned.
+/// @return last execution time in mocroseconds(us), or 0 if error.
```
Comment (ajkxyz) [inline include/dmp_dv.h]: *m**i**croseconds*

File: include/dmp_dv.h [hunk 3]
```diff
@@ -239,6 +239,12 @@ int64_t dmp_dv_cmdlist_exec(dmp_dv_cmdlist cmdlist);
 int dmp_dv_cmdlist_wait(dmp_dv_cmdlist cmdlist, int64_t exec_id);
 
 
+/// @brief Get the last execution time in microseconds of specified command.
+/// @param cmdlist Handle to command list, when NULL the error is returned.
+/// @return last execution time in microseconds(us), or 0 if error.
+uint64_t dmp_dv_cmdlist_get_last_exec_time(dmp_dv_cmdlist cmdlist);
```
Comment (ajkxyz) [inline include/dmp_dv.h]: Change return type to `int64_t` and comment to
`/// @return Last execution time in microseconds (us), or -1 on error.`

Comment (ZongHong-Lyu) [reply include/dmp_dv.h]: How about remain return in `uint64_t` type, but return `(uint64_t)-1` if error?

Comment (ajkxyz) [reply include/dmp_dv.h]: Better to change to `int64_t` as the range of `int64_t` is enough for microseconds and valid elapsed time `>= 0`.

Comment (ZongHong-Lyu) [reply include/dmp_dv.h]: Fixed. Kernel driver interface is also updated to use s64 type as returned execution time also.

**Pull request merged**